ui/component/or-mwc-components/src/or-mwc-input.ts

```diff
@@ -1203,6 +1209,10 @@ export class OrMwcInput extends LitElement {
                     }
 
                     if (!(this.type === InputType.RANGE && this.disableSliderNumberInput)) {
+                        if (type === InputType.FILTER) {
+                            outlined = true;
+                        }
+
```

Comment:
Same as comment at the top about `FILTER` being used for styling...just use the property:

```
<or-mwc-input outlined ... />
```

Reply:
Removed as I removed the type FILTER